docker/dxl/Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+dxlclient = "*"
+dxlmarclient = "*"
+dxltieclient = "*"
+virtualenv = "*"
```

Comment:
Are you sure you need this?

Reply:
I'm not sure about all the packages, but it's a copy from [demisto/dxl:latest](https://hub.docker.com/r/demisto/dxl/tags).
its an updated docker for dxl integration.
[#7571](https://github.com/demisto/etc/issues/7571)

Reply:
remove the virtualenv then
```suggestion
```